### ISSUE TITLE
Explore whether GitHub actions might be useful for some flict users

### ISSUE THREAD

**Issue by jeremiah** [vinland-technology/flict#153]
# Feature request
Simple GitHub action to demonstrate how we might provide flict as a GitHub action.

## Is your feature request related to a problem? Please describe.
An automated tool can be an efficent and easy way for users to test flict. Using GitHub "actions" may be a way to create such an automated tool and put it in front of a lot of potential users.

## Describe a tool that might help here

https://github.com/features/actions

**Comment by kkrolczyk** [comment]
Repo already uses Github Actions, so i guess this issue should be closed?

**Comment by jens-erdmann** [comment]
No. This issue is about providing a Github Action that employs flict and can be used in other projects.
